redis/server/clean.sls

```diff
@@ -0,0 +1,44 @@
+
+{%- from "redis/map.jinja" import redis_settings with context %}
+{%- set cfg_name          = redis_settings.cfg_name -%}
+{%- set install_from      = redis_settings.install_from -%}
+{%- set svc_name          = redis_settings.svc_name -%}
+{%- set port              = redis_settings.port -%}
+
+redis-server-service-clean:
+  service.dead:
+    {%- if install_from == 'source' %}
+    - name: {{ svc_name }}_{{ port }}
+    {%- else %}
+    - name: {{ svc_name }}
+    {%- endif %}
```

Comment:
It may be [better for readability to set a variable outside the state](https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#light-conditionals-and-looping) like `port`, `svc_name`, …

Reply:
Done.